admin/sql/create_tables.sql

```diff
@@ -8,4 +8,18 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE listens (
+  uid             VARCHAR NOT NULL,
+  year            INT,
+  month           INT,
+  day             INT,
+  id              INT NOT NULL,
+  artist_msid     UUID,
+  album_msid      UUID,
+  recording_msid  UUID,
+  json            VARCHAR,
+  PRIMARY KEY(uid, id)
```

Comment:
Also, it is customary to have the id column for joining purposes:

    id          SERIAL,

Please add that at the top of the table definition.

Next, we need to add more indexes. We'll need an index on uid and possibly on all of the _msid columns. Alastair?